vividus-plugin-web-app/src/main/resources/properties/profile/web/headless/chrome/profile.properties

```diff
@@ -1,2 +1,2 @@
 selenium.browser=chrome
-web.driver.CHROME.command-line-arguments=--headless --hide-scrollbars
+web.driver.CHROME.command-line-arguments=--headless=chrome --hide-scrollbars
```

Comment:
can it be a potential breaking change for end users? as per https://chromium.googlesource.com/chromium/src/+/833543a511a970f44fb074b9f30c6b6268e0be71 its completely different browser implementation

Reply:
see initial PR comment